test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1840,3 +1840,44 @@ def test_held_jobs_equiv_class(self):
 
         self.scheduler.log_match("Number of job equivalence classes: 1",
                                  max_attempts=10, starttime=self.t)
+
+    def test_queue_resav(self):
+        """
+        Test that jobs in queues with resources_available limits use queue as
+        part of the criteria of making an equivalence class
+        """
+
+        a = {'resources_available.ncpus': 2}
+        self.server.create_vnodes('vnode', a, 1, self.mom, usenatvnode=True)
+
+        attrs = {'queue_type': 'Execution', 'started': 'True',
+                 'enabled': 'True', 'resources_available.ncpus': 1,
+                 'Priority': 10}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, attrs, id='workq2')
+
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
+
+        a = {'queue': 'workq', 'Resource_List.select': '1:ncpus=1'}
+        J = Job(TEST_USER, a)
+        jid1 = self.server.submit(J)
+
+        a['queue'] = 'workq2'
+        J = Job(TEST_USER, a)
+        jid2 = self.server.submit(J)
+
+        J = Job(TEST_USER)
+        jid3 = self.server.submit(J)
```

Comment:
I don't like relying on the fact that PBS/PTL will add a default select to the job and would suggest that you add the select line explicitly.

Reply:
I meant to include it.  I just missed it, thanks for catching this.